codecoverage.py

```diff
@@ -91,10 +91,15 @@ def generate_info(grcov_path):
         else:
             ordered_files.append("ccov-artifacts/" + fname)
 
+    # Assume we're on a one-click loaner.
+    mod_env = os.environ.copy()
+    if os.path.isdir('/home/worker/workspace/build/src/gcc/bin'):
```

Comment:
There are other environment variables available, too. I don't remember off the top of my head, but something along the lines of TASK_ID or TASKCLUSTER_WORKER.